### DISCUSSION TITLE
Is there any way to pin the file metadata header to keep it from scrolling out of view?

### DISCUSSION THREAD

**Discussion by JetForMe** [community/community#124881] [Repositories]
### Select Topic Area

Question

### Body

When you're browsing code on the website (e.g. https://github.com/apple/swift-package-manager/blob/c965d5ab7bcb98baa6139ebe27d0e5765fcff183/Sources/Workspace/DefaultPluginScriptRunner.swift#L584-L586), unless you're scrolled to the top, the header containing the org and repo names, and tabs like Code, Issues, Wiki, etc., is hidden.

I find myself needing that header much more often than I need the extra few pixels of code view it affords when hidden. Is there any way to set a preference to prevent it scrolling away?

<img width="1320" alt="Shown" src="https://github.com/community/community/assets/401600/269e6b1e-ee22-44c3-9975-48c139fbcda1">
<img width="1320" alt="HiddenDueToScroll" src="https://github.com/community/community/assets/401600/e313a371-7cc2-4d99-8b54-13d5faa894b4">

If not, could such a thing be added? I find the current behavior really gets in my way.

**Comment by queenofcorgis** [comment]
Hey there @JetForMe - thanks for this feedback. There currently isn't a way, but I did a quick search and noticed there are several projects on GitHub that may be able to help with a workaround for this 🛠️ . I've also tagged this with the `Product Feedback` label so it can be reviewed by the appropriate team. 

Feel free to comment or edit your post to add any more information, useful details could include: use cases, relevant labels, desired outcomes, and any accompanying screenshots. 

As a member of the GitHub community, your participation is essential. While we can't promise that every suggestion will be implemented or get an individual response, we want to emphasize that your feedback is instrumental in guiding our decisions and priorities.
